StopDetailHeading.tsx

```diff
@@ -1,9 +1,33 @@
 import React from 'react';
+import PopoverMenu from './PopoverMenu';
 
 interface StopDetailHeadingProps {
   name: string;
+  stopId: number;
+  onStopAmenitiesClick: () => void;
 }  
 
-function StopDetailHeading({ name }: StopDetailHeadingProps) {
-  return <h2>{name}</h2>
+function StopDetailHeading({ name, stopId, onStopAmenitiesClick }: StopDetailHeadingProps) {
+  
+  const menuItems = [
+    {
+      name: 'Stop Schedule',
+      onSelect:  window.open('https://trimet.org/ride/stop_schedule.html?_LOCALE_=en&stop_id=${stopId}', '_blank'),
+      href: 'https://trimet.org/ride/stop_schedule.html?_LOCALE_=en&stop_id=${stopId}',
+      key: 0
```

Comment:
for lines 17 and 23, noticied we are hardcoding keys, shouldn't we be using [UUID](https://www.npmjs.com/package/uuid) instead?